src/Misc/layoutbin/en-US/strings.json

```diff
@@ -461,6 +461,7 @@
   "ProcessCompletedWithCode0Errors1": "Process completed with exit code {0} and had {1} error(s) written to the error stream.",
   "ProcessCompletedWithExitCode0": "Process completed with exit code {0}.",
   "ProcessExitCode": "Exit code {0} returned from process: file name '{1}', arguments '{2}'.",
+  "ProcessHandlerScriptArgsSanitized": "Detected characters in arguments that may not be executed correctly by the shell. Please escape special characters using circumflex (^). More information is available here: https://aka.ms/ado/75787",
```

Comment:
I'd delete "Please escape special characters using circumflex (^)."  As it is; its invite more questions which will be too long for an error message.  (e.g. what characters?  when do they need to be escaped)  Best to leave specific guidence for the documentation

Reply:
🔧